sanitize-markdown.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var he = require('he');
+var she = require('./she');
```

Comment:
this doesn't appear to be used, but for the sake of minimizing changes, I left the reference in still